worker-example/pom.xml

```diff
@@ -46,12 +46,29 @@
     <artifactId>worker-example-aggregator</artifactId>
     <version>1.4.0-SNAPSHOT</version>
     <packaging>pom</packaging>
-    <description>Builds Worker-Example projects together from a handy top level project.</description> 
+    <description>Builds Worker-Example projects together from a handy top level project.</description>
+
+    <!--
+    The dependency management section is a mechanism for centralizing dependency information.
+    The CAF worker framework is specified.
+-->
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.github.workerframework</groupId>
+                <artifactId>worker-framework</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
 
     <modules>
         <module>worker-example</module>
         <module>worker-example-shared</module>
         <module>worker-example-container</module>
+        <module>worker-example-container-fs</module>
         <module>worker-example-message-builder</module>
     </modules>
 
```

Comment:
I see below all the folders have been renamed from "worker-example-container" to "worker-example-container-fs" yet we still have a worker-example-container module, is this also correct?

Is this part of having the files for the containers only in one place?